src/main/kotlin/com/pubnub/api/endpoints/objects/internal/ReturningCustom.kt

```diff
@@ -45,8 +54,17 @@ data class ReturningCustom @Deprecated("use Include", ReplaceWith("Include(inclu
     }
 }
 
-@Deprecated("use Include", ReplaceWith("Include"))
-data class ReturningChannelDetailsCustom @Deprecated("use Include", ReplaceWith("Include(includeCustom = includeCustom, includeChannelDetails = includeChannelDetails)"))constructor(
+@Deprecated(
+    "use IncludeQueryParam",
+    ReplaceWith("IncludeQueryParam", "com.pubnub.api.endpoints.objects.internal.IncludeQueryParam")
+)
+data class ReturningChannelDetailsCustom @Deprecated(
+    "use IncludeParam",
```

Comment:
seems that should be "use IncludeQueryParam" instead of "use IncludeParam"

Reply:
yes, but I realised that we can actually remove this completely